buildroot.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
+is_arg=false
 
-docker-compose pull
-docker-compose run --rm buildroot
+for args in "$@"
+do
+  is_arg=true
 
+  if [ "$args" = "--help" ]; then
```

Comment:
this is not the right way to do this. See example here: https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
using a switch statement